roles/common/tasks/oh-my-zsh.yml

```
@@ -0,0 +1,14 @@
+---
+
+- name: Download oh-my-zsh
+  git:
+    repo: 'https://github.com/ohmyzsh/ohmyzsh.git'
+    dest: "/home/{{ user }}/.oh-my-zsh"
+    version: master
```

Add this option to prevent colliding operations with the oh-my-zsh command

```suggestion
    version: master
    update: no
```